tee.js

```diff
@@ -49,7 +50,9 @@ function alwaysTrue () {
   return true
 }
 
-module.exports = tee
+module.exports = transport
+
+module.exports.tee = tee
```

Comment:
I suspect this change is because the transport mechanism requires it to be the main export, right? this also means that this is a breaking change for Node.js users. Which I guess is fine too

Reply:
yep, exactly. I did change the docs to reflect this change.

Reply:
can I ask you do to do one thing? I'm ok with this change but for consistency can we check how other pino transports have handled this when introducing support for v7? I suspect they had to do something similar and we should make sure we do it in the same way

Reply:
loks like pino pretty did basically the same thing [here](https://github.com/pinojs/pino-pretty/commit/72808da44b708bbc3b50fbb3ecddb89f706a458c#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R209)